### ISSUE TITLE
[Feature Request] Display pull requests

### ISSUE THREAD

**Issue by mustafa01ali** [github/android#663]
Would be nice if pull requests can be reviewed and merged on the go.

**Comment by kramimus** [comment]
Agreed, a 5th tab on the home screen to view open pull requests without hunting through the news feed would be great.

**Comment by soul9** [comment]
Pull requests' details can not be displayed currently at all. You can press on the pr and you get a blank page. This app is useless.

**Comment by stilez** [comment]
The "blank page" issue seems to be some kind of issue with permissions - it won't load issues unless the user has write access or something weird, judging by logcat.  So you get a blank page when you try to view an issue where you put in a PR because you don't have commit access to that repo or something.

**Comment by marc0der** [comment]
+1 for a feed op pull requests, as well as the ability to merge them on the go.
